src/components/vive-controls.js

```diff
@@ -174,6 +177,13 @@ module.exports.Component = registerComponent('vive-controls', {
     controllerObject3D.position.set(0, -0.015, 0.04);
   },
 
+  onAxisMoved: function (evt) {
+    if (evt.detail.axis[0] === 0 && evt.detail.axis[1] === 0) {
```

Comment:
Can we do this a one liner? if (evt.detail.axis[0] === 0 && evt.detail.axis[1] === 0) { return; }`